environment.yml

```diff
@@ -54,7 +57,7 @@ dependencies:
   - coverage
   - cython >=0.25.2
   - scikit-learn
-  - scipy
+  - scipy <1.11
```

Comment:
Why do we add this?

Reply:
Thanks for catching this. I believe I originally added this because of descriptastorus (which was incompatible with later versions of scipy) but it might not be required. I will remove it and we can see what happens in the CI.